wdqs/0.3.10/Dockerfile

```diff
@@ -15,10 +15,12 @@ FROM openjdk:8-jdk-alpine
 # Install gettext for envsubst command, (it needs libintl package)
 # Install curl for the loadData.sh wdqs script (if someone needs it)
 RUN set -x ; \
-    apk --no-cache add bash=\<4.5.0 gettext=\<0.19.8.2 libintl=\<0.19.8.2 curl=\<7.64.999
+    apk --no-cache add bash=\<4.5.0 gettext=\<0.19.8.2 libintl=\<0.19.8.2 curl=\<7.64.999 sudo=\<1.9
```

Comment:
Is it possible to just use plan `su` so we don't need to add this dependency? I don't think blazegraph needs to be on an interactive TTY so maybe we could just use `su -c` ?

Reply:
su is not super great in normal computer use but I understand this is different in containers, let me fix it then.

Reply:
su doesn't work in this case, "runuser" might.

Reply:
> su doesn't work in this case

What's the problem?
 sudo -u doesnt work? :(

Maybe we could / should do something like https://github.com/elastic/dockerfiles/blob/v7.5.1/elasticsearch/bin/docker-entrypoint.sh#L7-L16

Reply:
sudo -u needs "sudo" package to be installed in the alpine docker images